src/c++/perf_analyzer/test_load_manager_base.h

```diff
@@ -60,17 +60,18 @@ class TestLoadManagerBase {
       bool is_decoupled_model)
       : params_(params)
   {
-    // Must reset this global flag every unit test.
-    // It gets set to true when we deconstruct any load manager
-    // (aka every unit test)
-    //
-    early_exit = false;
-
     stats_ = std::make_shared<cb::MockClientStats>();
     factory_ = std::make_shared<cb::MockClientBackendFactory>(stats_);
     parser_ = std::make_shared<MockModelParser>(
         is_sequence_model, is_decoupled_model);
   }
+
+  ~TestLoadManagerBase()
+  {
+    // Reset early_exit in case any test sets it to true during execution.
+    early_exit = false;
```

Comment:
Were you observing it to be sticky across tests?

Reply:
Yes.